scalding-core/src/main/scala/com/twitter/scalding/FileSource.scala

```diff
@@ -419,9 +419,22 @@ abstract class FixedPathSource(path: String*) extends FileSource {
   def localPaths = path.toList
   def hdfsPaths = path.toList
 
+  override def hdfsWritePath = stripTrailingStar(super.hdfsWritePath)
+
   override def toString = getClass.getName + path
   override def hashCode = toString.hashCode
   override def equals(that: Any): Boolean = (that != null) && (that.toString == toString)
+
+  /**
+    * Strip trailing '*' from the path string.
+    */
```

Comment:
can you add a link to the `TimePathedSource` method `writePathFor`?

It seems like that code is dropping the trailing `/`. Should we not do the same thing here? I think that was done to accomodate local mode which writes to one file. Hadoop seems to create a directory in any case.

Also, can we make that code use the same style `.dropRight(2)` in that case?